### DISCUSSION TITLE
Custom headers (specifically cookies)

### DISCUSSION THREAD

**Discussion by traffisco** [flareact/flareact#199] [Ideas]
So I need to set server side cookies in first response.

What do you think about adding support for custom headers?  It could be implemented easily. 
Only question is about the cookie header, should it be left to the user to create valid headers or should some more cookies specific solution should be made?

**Comment by tj-kev** [comment]
NextJS does this from a [config file](https://nextjs.org/docs/api-reference/next.config.js/headers) but only for known header values.

For dynamic headers like cookies it think possibly left to the user to create whatever they want?

**Reply by traffisco** [reply]
@tj-kev any plans merging this?

**Reply by tj-kev** [reply]
Yeah sorry, merged into canary now.

**Comment by traffisco** [comment]
True. See #200
